src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,125 @@
+@import "utils/variables";
+
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  max-width: 944px;
+  box-sizing: content-box;
+  &__card {
+    margin: 0 48px 46px 0;
+    padding: 32px 16px 16px;
+  }
+}
+
+.card {
+  max-width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+  &__link {
+    display: block;
+  }
+  &__img {
+    display: block;
+    max-width: 100%;
+    height: 134px;
+    padding: 0 3px;
+    margin-bottom: 40px;
+    text-align: center;
+    transition: transform 0.5s;
+    &:hover {
+      transform: scale(1.1);
+    }
+  }
+  &__title {
+    display: block;
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-text-color;
+  }
+  &__product-code {
+    margin-bottom: 16px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 140%;
+    color: #616070;
+  }
+  &__stars {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+  &__stars-review {
+    position: relative;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $main-text-color;
+    transition: 0.5s color;
+    &:after {
+      content: "";
+      position: absolute;
+      bottom: 0;
+      left: 0;
+      height: 1px;
+      width: 100%;
+      background-color: $link-color;
+      transform: scaleX(0);
+      transform-origin: right;
+      transition: 0.5s transform;
+    }
+    &:hover {
+      color: $link-color;
+      &:after {
+        transform: scaleX(1);
+        transform-origin: left;
+      }
+    }
+  }
+  &__prices {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+  &__price {
+    color: #616070;
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 150%;
+  }
+  &__value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 112%;
+  }
+  &__button {
+    display: flex;
+    justify-content: center;
+    width: 166px;
+    height: 40px;
+    padding: 11px 0;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    color: #fff;
+    background: $link-color;
+    border-radius: 5px;
+    text-transform: uppercase;
+    border: 1px solid transparent;
+    transition: 1s all;
```

Comment:
don't use it like that, because it's the same as transition-duration: 1s;
It's too hard for the browser
![image](https://user-images.githubusercontent.com/91411692/222698665-820ac0ec-21a0-4166-9b27-adfea7b6a0cd.png)

```suggestion
    transition: ... $transitionDuration;
```